config/secrets.production.yml

```diff
@@ -0,0 +1,23 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 05d38731bbf4e8420b2d1b0b861f8b657d8765120eb25636fe3fc27d074b3be98db6d0d9d002a9b521e429d4734058b3b266eb55de4dbcc6e05aadb96e70fa3d
+
+test:
+  secret_key_base: c3394eca75bcae833976db5ebb61e4717e5e20d8b7362835332c35227d9846afbab26a0c833315e0b88a540f0d54d5a253d368930e495b5cf037daf4012c1590
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
+  secret_key_base: c3394eca75bcae833976db5ebb61e4717e5e20d8b7362835332c35227d98$
```

Comment:
Sorry, what? You shouldn't place any secret data for production in a public repository.